portfolio.html

```diff
@@ -12,14 +12,17 @@
 	 	<nav class="navholder">
 			<div class="logo"><a href="#"><img class="logoimg" alt="" src=></a></div>
```

Comment:
rather than add the links to divs, I think you can just assign the class to the <a>
    It doesn't change anything visually but it reduces you code a bit

Reply:
Will keep it in mind for future reference